packages/lib/services/apiKey.ts

```diff
@@ -7,21 +7,22 @@ import { getHash } from "../crypto";
 import { createHash, randomBytes } from "crypto";
 import { DatabaseError, InvalidInputError, ResourceNotFoundError } from "@formbricks/errors";
 import { cache } from "react";
+import { hasUserEnvironmentAccess } from "../../../apps/web/lib/api/apiHelper";
```

Comment:
that import seems wrong. we should move the apiHelper to the lib package.

Reply:
Please move to `packages/lib/helpers/api`. Currently still thinking about the notation. I will ask in Slack.